style.css

```diff
@@ -0,0 +1,105 @@
+* {
+    box-sizing: border-box;
+}
+a {
+    color: inherit;
+    text-decoration: inherit;
+}
+body {
+    margin: 0;
+    font-size: 20px;
+    font-family: 'Josefin Sans', sans-serif;
+    color: #aca9a5;
+    background-color: #8d7e55;
+    height: 2000px;
+}
+nav {
+    position: fixed;
+    width: 100%;
+}
+.nav_container {
+    color: #8d7e55;
+    overflow: auto;
+    margin: 0 auto;
+    width: 95vw;
+    padding: 20px;
+    border-bottom: solid 2px #4b3341;
+    font-size: 1rem;
+}
+.burgerselector {
+    /*Currently an ugly button, I know right But this will change for sure yo*/
+    height: 85px;
+    width: 85px;
+    display: none;
+}
+.logo_name {
+    font-size: 1.25rem;
+    float: left;
+    margin-top: 25px;
+}
+.logo {
+    position: absolute;
+    left: 35px;
+    top: 110px;
+    height: 8vw;
+    width: 8vw;
+    min-width: 125px;
+    min-height: 125px;
+}
+.nav_links {
+    float: right;
+  }
+.nav_buttons {
+    display: inline-block;
+    width: 100px;
+    text-align: center;
+    padding: 12px 5px;
+    -webkit-border-radius: 7px;
+    -moz-border-radius: 7px;
+    border-radius: 7px;
+    margin-left: 20px;
+    margin-top: 12px;
+    overflow: auto;
+    border: 2px solid transparent;
+}
+.home_page {
+    background-image: url("BackgroundHome.jpg");
```

Comment:
probably best to keep all images in an image folder in your project to keep everything neat and indexed